fixtures/u2_class_this.go

```diff
@@ -0,0 +1,8 @@
+package fixtures
+
+type Testcls1 struct {
+	a int
+}
+func (this Testcls1) Testfnc1() {
```

Comment:
and a second test to show that `this` can have different name

Reply:
I've updated the spec to include this.